pages/price-feeds/fetch-price-updates.mdx

```diff
@@ -0,0 +1,76 @@
+import { Callout } from "nextra/components";
+
+# How To Fetch Price Updates
+
+The following guides explain how to fetch price updates to update Pyth price feeds on-chain.
+Price updates can be submitted to the Pyth Price Feeds contract to update the on-chain price.
+These price updates are fetched from [Hermes](/price-feeds/how-pyth-works/hermes) which are verified on-chain.
+Please see [What is a Pull Oracle?](/price-feeds/pull-updates) to learn more.
+
+There are three different ways to fetch price updates:
+
+1. [REST API](#rest-api)
+1. [SDK](#sdk)
+1. [Websocket](#websocket)
```

Comment:
I think the order should be REST API, websocket, SDK. (The SDK is a wrapper around the other two)